paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -2682,4 +2693,33 @@ public InlineElement getDescription() {
             return text(description);
         }
     }
+
+    /** Time field type for record level expire. */
+    public enum TimeFieldType implements DescribedEnum {
+        SECOND(
+                "second",
+                "10-bit timestamp which indicates the number of seconds from 00:00:00 UTC on January 1, 1970 to the present, excluding milliseconds."),
+
+        MILLISECOND(
+                "millisecond",
```

Comment:
`millis-long`

Reply:
Fixed in https://github.com/apache/paimon/commit/20a186ed4c7abbf08f9b89ae33f74a5075d068dd